charts/gradle-cache/values.yaml

```diff
@@ -0,0 +1,137 @@
+# Override name of the chart
+nameOverride: ""
+# Full name of the chart.
+fullnameOverride: ""
+
+# Image pull secret for private images
+imagePullSecrets: []
+image:
+  repository: gradle/build-cache-node
+  tag: ""
+  pullPolicy: IfNotPresent
+
+replicaCount: 1
+
+configSecret:
+  # Specifies whether a secret should be created
+  create: true
+  # Name of the secret to use; if not set and create is true, a name is generated using the fullname template
+  name: 
+  # Content of 'config.yaml'
+  content: |-
+    version: 5
+    cache:
+      accessControl:
+        anonymousLevel: readwrite
+      freeSpaceBufferSize: 2048
+      maxEntryAgeInHours: 720
+      maxArtifactSize: 1024 
+      targetSize:
+        type: fixed
+        size: 10000
+    uiAccess:
+      type: open
+    registration: null
+
+externalSecret:
+  create: false
+  name: myExternalSecretName
+  refreshInterval: 1h
+  secretStoreName: existedClusterSecretStoreName
+  secretName: mySecretName
+  remoteKey: azureKeyvaultSecretKey
+
+externalSecretStore:
+  create: false
+  name: secretStoreName
+  vaultUrl: https://MY-KEY-VAULT.vault.azure.net/
+
+args:
+  - start
+  - --no-warn-anon-cache-write
+  - --no-warn-anon-ui-access
+
+storage:
+  storageClass: storageClassName
+  size: 10Gi
```

Comment:
Were you able to test if this is enough storage?

Reply:
This is default values for chart. It will be overridden upon deployment